docs/docs/contribute/software/add-new-metric-provider.md

```diff
@@ -0,0 +1,127 @@
+---
+comments: true
+---
+
+# Adding a New Metrics Provider for Placeholder Endpoint
+
+In this guide, we will create a placeholder provider.
+The following steps are a starting point to create your own custom provider:
+
+<!-- markdownlint-disable MD029 -->
+<!-- markdownlint-disable MD007 -->
```

Comment:
why is this needed ? Let's rather fix the problems than adding md linter ignorers

Reply:
@StackScribe is their a way for us to remove the md linter ignorers completely?

Reply:
I think @mowies fixed the linters so that it is no longer necessary to disable them -- and after you went to all that trouble!  Try removing them and look at the rendered text to be sure that it is correct.

Reply:
Alright I will create a separate PR for this.